subsys/bluetooth/controller/ll_sw/ull_filter.c

```diff
@@ -138,10 +140,29 @@ static uint32_t filter_remove(struct lll_filter *filter, uint8_t addr_type,
 			   uint8_t *bdaddr);
 #endif /* !CONFIG_BT_CTLR_PRIVACY */
 
-static void filter_insert(struct lll_filter *filter, int index, uint8_t addr_type,
-			   uint8_t *bdaddr);
+static void filter_insert(struct lll_filter *filter, int index,
+			  uint8_t addr_type, uint8_t *bdaddr);
 static void filter_clear(struct lll_filter *filter);
 
+#if defined(CONFIG_BT_CTLR_SYNC_PERIODIC)
+#define PL_ADDR_MATCH(type, addr) \
+		(pl[i].taken && \
+		(pl[i].id_addr_type == (type & 0x1)) && \
```

Comment:
Do we have a #define for the type values, instead of 0x1?

Reply:
`0x01` is used as 1-bit mask, as `id_addr_type` is 1-bit size.